### ISSUE TITLE
"Black queen" by Dreamer can not be played

### ISSUE THREAD

**Issue by hrw** [cmatsuoka/libxmp#124]
Dreamer's module "Black queen" can not be played. First patterns are played and that's all. 

Module is doing several nasty things with Protracker commands which could be the problem.

ftp://ftp.modland.com/pub/modules/Protracker/Dreamer/black%20queen.mod

**Comment by cmatsuoka** [comment]
Indeed. I also tried with with oxdz (a new proof-of-concept player I wrote in Rust) and it plays for 1:57, which is the correct duration, but it still sounds ugly. Libxmp breaks it in many subsongs and plays only the first one.

I'll use this as a reference of how it's supposed to sound: https://www.youtube.com/watch?v=1fE3N8_UWdY

Perhaps an option to disable or tweak subsong heuristics could help.

**Comment by LossyDragon** [comment]
FWIW, holding the module in a loop allows it to play fine.
